docs/Privacy/Tessera/Usage/Logging.md

```diff
@@ -0,0 +1,71 @@
+Messages are written to the logs using these rules for the log level:
+
+* ERROR: system failures or situations that require some action to ensure correct operation of the system.
+* WARN: notifications that don't require immediate action or that are indications that a transaction failed.
+* INFO: information message to allow investigation of issues or to provide reassurance that the system is operating correctly. 
+* DEBUG: very verbose logging to assist with investigation of issues
+
+The log level is written out in uppercase as part of the log message, this can be used for alert monitoring.
+
+## Errors
+Below is a non-exhaustive list of error messages and suggested actions. Braces '{}' indicate where further detail of the root cause is logged as part of the message.
+
+Message | Cause
```

Comment:
Suggestion: to change the table to 3 columns with `Message | Cause | Action` so that the action is very clear.

Reply:
Adding a third column has a negative impact on the formatting, however the steps have been emphasised so that they are easier to identify